fhir_ig_authoring.md

```diff
@@ -4,37 +4,40 @@ This guide assumes you have a basic understanding of FHIR and profiling. If not,
+In FHIR, an instruction on how to build resources for a specific use case is called an *Implementation Guide* (IG). An IG usually consists of text explaining the use case and how FHIR plays a role,
+profiles that describe how FHIR resources should look like in the context of this project ( if you are not familiar with what FHIR profiles are, read e.g. the introduction [here](https://www.hl7.org/fhir/profiling.html) ) and some terminology resources like ValueSets, ConceptMaps or CodeSystems that tell implementers which codes to use when in the context of the project covered by the IG.
 
 ## How to extend the data model
 
-It may become necessary to extend the data model, e.g. to add a new criteria tat can be searched for. In this case, you should first answer the following questions:
+It may become necessary to extend the data model, e.g. to add a new data elements that can be searched for. In this case, you should first answer the following questions:
 
-    *1. Is the addition part of the core GBA IG, the Oncology IG (see below), or should it be a new module?* Is it something all or most participatimg biobanks must or may want to use 
-    or is it only relevant for a subgroup / subproject? In case of subprojects, you may want to create a new IG that depends on the core GBA IG and has profiles that inhert from the core profile (for more info on profile inheritance, read the corresponding section below or https://www.hl7.org/fhir/profiling.html#reslicing).
-    *2. Do other IGs already cover this scope?* A web search for "FHIR + Topic" may find work that other people already have done on this subject. If nothing turns up, ask on the #implementers stream of chat.fhir.org (accounts are free and a must have for everyone working with FHIR). If you find a fitting IG thats seems well-maintained, use it e.g. by adding a short section with a description (e.g. which specifics profiles out of the IG to use when)  and a link to the core IG. Also add a formal dependency to the package the IG corresponds to (in Simplifier: Package Tab of project corresponding to the IG, with IG Publisher Templates the Package Id and version are usually in the website footer) to the core IG (see https://fshschool.org/docs/sushi/configuration/#recommended-configuration). Using parts of other IGs instead of defining your own profiles helps interoperability and is in the spirit of FHIR (and also usually less work).
+*1. Is the addition part of the core GBA IG, the Oncology IG (see below), or should it be a new module?* Is it something all or most participating biobanks must or may want to use, or is it only relevant for a subgroup / subproject? In case of subprojects, you may want to create a new IG that depends on the core GBA IG and has profiles that inherit from the core profile (for more info on profile inheritance, read the corresponding section below or <https://www.hl7.org/fhir/profiling.html#reslicing>).

```

Comment:
Markdown formatting still doesn't work here. Please remove the `*`: https://daringfireball.net/projects/markdown/syntax#list